ivy/functional/ivy/experimental/norms.py

```diff
@@ -140,3 +140,44 @@ def instance_norm(
         track_running_stats=track_running_stats,
         out=out,
     )
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+def lp_normalize(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    p: int,
```

Comment:
I don't see why p should be int, all frameworks seems to accept floats as long as they're >0